doc/generated/pages/architecture/transports/pipeline.html

```diff
@@ -212,13 +212,15 @@ <h2>Segment parser</h2>
 <li>whether the segment should be offseted when decoded and by what amount</li>
 <li>the decodable data (which can be wrapped in a container e.g. subtitles in an
 ISOBMFF container).</li>
+<li>the attached protection information and data to be able to decrypt that
```

Comment:
Unexpected text

Reply:
Done. `doc/generated` is not updated anymore. We will only do it at release time.